web/src/lib/components/shared-components/side-bar/side-bar-button.svelte

```diff
@@ -1,5 +1,7 @@
 <script lang="ts">
 	export let title: string;
+	// TODO: why `any` here? There should be a expected type for this
```

Comment:
`logo` is a Svelte component, it is from the library https://www.npmjs.com/package/svelte-material-icons

I think it can be set as `HTMLElement` though I haven't tested it yet.